index.bs

```diff
@@ -994,6 +994,22 @@ Consider adding such features only in cases when the overall user experience is
 A canonical example of this is blocking rendering in order to download and process a stylesheet. 
 The alternative user experience is a flash of unstyled content, which is undesirable.
 
+<h3 id="html-idl-must-by-synced">Keep attributes in sync</h3>
+
+If your feature introduces new attributes,
+try to ensure the attributes are synchronized.
+
+This means any new content attribute
+should always have a corresponding IDL attribute with the same name,
```

Comment:
```suggestion
should have a corresponding IDL attribute with the same name,
```

Reply:
Now that you merged my suggestion, we are basically saying the same thing twice. We should merge paragraphs 1 and 2.